### ISSUE TITLE
Add slave_port attribute to metric "redis_connected_slave_offset"

### ISSUE THREAD

**Issue by redzippo1** [oliver006/redis_exporter#219]
Hi oliver006

Thank you for take redis_exporter for us.
1 Now the metric redis_connected_slave_offset do not have slave_port attribute, I think it is important too.
2 Now the metric redis_instance_info print twice in http api result, is it a bug?

```
# HELP redis_instance_info Information about the Redis instance
# TYPE redis_instance_info gauge
redis_instance_info{addr="101.255.24.14:8391",alias="f3bok3wU",os="",redis_build_id="",redis_mode="",redis_version="",role=""} 1
redis_instance_info{addr="101.255.24.14:8391",alias="f3bok3wU",os="Linux 3.10.0-229.el7.x86_64 x86_64",redis_build_id="2c78cbb5af4c3382",redis_mode="cluster",redis_version="3.2.11",role="slave"} 1
redis_instance_info{addr="101.255.24.14:8392",alias="f3bok3wU",os="",redis_build_id="",redis_mode="",redis_version="",role=""} 1
redis_instance_info{addr="101.255.24.14:8392",alias="f3bok3wU",os="Linux 3.10.0-229.el7.x86_64 x86_64",redis_build_id="2c78cbb5af4c3382",redis_mode="cluster",redis_version="3.2.11",role="master"} 1
```

I hope to help you do this PR ：）

**Comment by oliver006** [comment]
1) Yeah, that sounds important. I don't have time to add this right now but a PR is welcome and will be reviewed and merged as soon as possible.

2) It prints once for each port `"101.255.24.14:8391"` and `"101.255.24.14:8392"` - looks like you're running and scraping two exporters?

**Comment by redzippo1** [comment]
Thank you for reply.
For the question 2, I found the reason is: function extractInfoMetrics called twice in once scrape.
The first time is for the "CLUSTER INFO", and second time is for the "INFO ALL".
And the code "e.metrics["instance_info"].WithLabelValues().set()" will called twice.

Key code:

```
if isClusterEnabled {
		if clusterInfo, err := redis.String(doRedisCmd(c, "CLUSTER", "INFO")); err == nil {
			e.extractInfoMetrics(clusterInfo, addr, e.redis.Aliases[idx], scrapes, dbCount, false)

			// in cluster mode Redis only supports one database so no extra padding beyond that needed
			dbCount = 1
		} else {
			log.Errorf("Redis CLUSTER INFO err: %s", err)
		}
	} else {
		// in non-cluster mode, if dbCount is zero then "CONFIG" failed to retrieve a valid
		// number of databases and we use the Redis config default which is 16
		if dbCount == 0 {
			dbCount = 16
		}
	}

	e.extractInfoMetrics(infoAll, addr, e.redis.Aliases[idx], scrapes, dbCount, true)
```

**Comment by oliver006** [comment]
Thanks @redzippo1 - I'll try to have a look at the PR as soon as possible (next few days) and get back to you!

**Comment by oliver006** [comment]
Closed by #220
